variables.tf

```diff
@@ -94,6 +94,16 @@ variable "lambda_memory" {
   default     = 128
 }
 
+variable "ignore_delete_errors" {
+  description = <<-EOF
+    Ignore CloudFormation stack errors from deletion events.
+
+    Setting this to true means that leftover Subscription Filters could remain.
+  EOF
+  type        = bool
+  default     = false
```

Comment:
Would `true` be a better default? Seems odd that you'd have to stall to find out what this option does.

Reply:
I think it's a more usable default. Users just might be surprised by it if we don't clean up properly.